test/io/readfile/f2.txt

```diff
@@ -0,0 +1,3 @@
+Ravi Shastri has said that Virat Kohli has given his side of the story on his exit as India white-ball captain, and now BCCI president Sourav Ganguly needs to give his version.Speaking at The Indian Express e.Adda, the former India head coach added that the captaincy change could have been handled better with “good communication.”Days after Sourav Ganguly had said that he had personally told Virat Kohli not to give up the T20I captaincy, Kohli said that there had been no communication from the board once he had decided to relinquish leadership in the shortest format in September.Regarding the ODI captaincy, Kohli said he had been informed that the selectors were removing him only 90 minutes before the selection committee meeting to pick the Test squad for the South Africa tour earlier this month.“With good communication, this can be handled better,” Ravi Shastri said. “Virat has given his side of the story. It needs the board president to give his side of the story. It’s not about who’s lying and who’s not. It’s not right to jump into there until you know what was the actual conversation.”Shastri added that once Kohli had decided he did not want to continue as T20I captain, there could have been only one leader for the two white-ball formats, where Rohit Sharma has now taken over.My statement made Ashwin do something different: Shastri R Ashwin had said recently that he felt “absolutely crushed” when Shastri stated after the Sydney Test in January 2019 that Kuldeep Yadav was India’s No.1 overseas spinner.Shastri responded that he was happy if his statement had been able to goad Ashwin into becoming fitter. “The way Ashwin bowled in 2019 and the way he has bowled in 2021 is chalk and cheese,” Shastri said.
```

Comment:
Is this copyrighted text? If you are just looking for some input text you can use the copy of Moby Dick we already have in our test system -- `test/types/string/psahabu/perf/moby.txt`. You can make a symbolic link to it with e.g. `ln -s ../../types/string/psahabu/perf/moby.txt moby.txt`.

Reply:
@mppf  Ok

Reply:
@mppf  I shall place the moby.txt in the test directory for our tests. Will that be a problem?

Reply:
the symbolic link can be in the directory with the tests